extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -259,6 +286,22 @@ export class AmpStoryQuiz extends AMP.BaseElement {
    * @private
    */
   handleOptionSelection_(optionEl) {
+    // TODO(jackbsteinberg): FIND A BETTER METHOD OF SETTING AND GETTING IDS, THIS IS REALLY FLIMSY LOL
+    // update the store service
+    this.storeService_.dispatch(Action.SET_QUIZ_INFO, {
```

Comment:
Here you can just use the variable service directly to update the variable without having to go through the store service, just import the variable service and use the `onVariableUpdate` method.

See example:
https://github.com/ampproject/amphtml/blob/57a8e3a03d19d5dfb3bff274605036aecc24df4e/extensions/amp-story/1.0/bookend/amp-story-bookend.js#L567-L582

Reply:
Ah thanks for the example, it makes sense that `getVariableService` always creates a reference to the same variable service.

Fixed.